src/util/irep.h

```diff
@@ -18,7 +18,9 @@ Author: Daniel Kroening, kroening@kroening.com
 
 #define SHARING
 // #define HASH_CODE
-// #define NAMED_SUB_IS_FORWARD_LIST
+#ifndef NAMED_SUB_IS_FORWARD_LIST
+#  define NAMED_SUB_IS_FORWARD_LIST
```

Comment:
I'm probably being thick here but why not just `#define` it?

Reply:
This ensures compatibility with any builds (such as our CI!) using `-DNAMED_SUB_IS_FORWARD_LIST` on the compiler command line.